apps/condo/public/robots.txt

```diff
@@ -0,0 +1,5 @@
+User-agent: *
+Allow: /auth/
+
+User-agent: *
```

Comment:
It seems, like that second `User-agent` declaration is redundant.
```suggestion
```

Reply:
I'm not sure. Lots of examples contain robots like this. Even if it's a redundant block, I don't think it will create any problems.